src/opnsense/scripts/suricata/installRules.py

```diff
@@ -64,6 +64,10 @@
                     if rule[i] != '#':
                         break
 
+                #consume an extra space before generating altered rule
+                if rule[i] == ' ':
```

Comment:
it's probably better to move it up to the existing loop, such as `if rule[i]  not in ['#', ' ']:`

Reply:
Thanks! Much nicer!

Reply:
@Tra5is thanks for taking care of this